develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/03-upgrading-the-liferay-maven-environment.markdown

```diff
@@ -0,0 +1,107 @@
+# Upgrading the Liferay Maven Environment
+
+If you're an avid Maven user, and have been using it for Liferay Portal 6.2
+project development, you must upgrade your Maven environment to be compatible
+with @product-ver@ development. There are two main parts of the Maven
+environment upgrade process that you must address:
+
+- Upgrading Liferay Maven Plugins
+- Upgrading Liferay Maven Artifacts
+
+The changes for both of these topics only occur in your parent POM and any
```

Comment:
I would leave this sentence out, not sure it tells them anything they don't know :)